build/charts/antrea/Chart.yaml

```diff
@@ -5,7 +5,7 @@ displayName: Antrea
 home: https://antrea.io/
 version: 0.0.0
 appVersion: latest
-kubeVersion: ">= 1.16.0-0"
+kubeVersion: ">= 1.19.0-0"
```

Comment:
It's pretty minor but this change (and the corresponding one in build/charts/flow-aggregator/Chart.yaml) should be in a sepatate PR (that we could even backport to release-2.0).

Reply:
Alright reverting this.

Reply:
You need to run `make` again under `build/charts` after this change.
Also, could you open a new PR to update the kubeVersion requirement for existing charts?

Reply:
ping for this